packages/package-json-lint/src/rules/utils/require-version-set.js

```diff
@@ -0,0 +1,20 @@
+const semver = require('semver');
```

Comment:
nitpick - i actually think this fits fine under the `rules` directory, rather than a `utils` sub-directory. We could just not export it in the index file. What do you think?